From 3a87cc0dadef1ce221d0b65a5a1f2c0dc24fa70c Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@st.com>
Date: Sat, 2 May 2020 16:47:45 +0200
Subject: [PATCH 0134/1695] plat-stm32mp1: rcc header can be included in
 assembly source file

Fix stm32mp1_rcc.h so that it can be included in an assembly source
file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 94e6dfc83b24a88fb3af60e03b5e1b69c4dfe9b1 plat-stm32mp1: rcc header can be included in assembly source file]
Change-Id: I026b6d353b68f9bed370e7addbfb4294fccb1201
Signed-off-by: Etienne Carriere <etienne.carriere@st.com>
---
 core/include/drivers/stm32mp1_rcc.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/include/drivers/stm32mp1_rcc.h b/core/include/drivers/stm32mp1_rcc.h
index b820a7c26..99c65a1a0 100644
--- a/core/include/drivers/stm32mp1_rcc.h
+++ b/core/include/drivers/stm32mp1_rcc.h
@@ -6,8 +6,6 @@
 #ifndef __DRIVERS_STM32MP1_RCC_H__
 #define __DRIVERS_STM32MP1_RCC_H__
 
-#include <io.h>
-#include <stdbool.h>
 #include <util.h>
 
 #define RCC_TZCR			0x00
@@ -550,6 +548,8 @@
 #define DT_RCC_SECURE_CLK_COMPAT	"st,stm32mp1-rcc-secure"
 
 #ifndef __ASSEMBLER__
+#include <io.h>
+#include <stdbool.h>
 #include <stm32_util.h>
 
 static inline bool stm32_rcc_is_secure(void)
-- 
2.47.2

